script.js

```diff
@@ -0,0 +1,39 @@
+const inputBox = document.getElementById("input-box");
+const listContainer = document.getElementById("list-container");
+
+//to add task through the input field into list//
+function addTask(){
+    if(inputBox.value === '') {
+        alert("You must write something");
+    }
+    else{
+        let li = document.createElement('li');
+        li.innerHTML = inputBox.value;
+        listContainer.appendChild(li);
+        let span = document.createElement('span');
+        span.innerHTML = "\u00d7";
+        li.appendChild(span);
+    }
+    inputBox.value = '';
+    getData();
+}
+//when any tasked is click, to reflect done or remove//
+listContainer.addEventListener('click', function(e) {
+    if(e.target.tagName ==='LI'){
+        e.target.classList.toggle('checked');
+        getData()
+    }
+    else if(e.target.tagName ==='SPAN'){
+        e.target.parentElement.remove();
+        getData()
+    }
+})
+//to store the task input so when browser is refreshed, it keep it//
+function getData(){
+    localStorage.setItem('data', listContainer.innerHTML);
```

Comment:
Ok i get that you want to save the data here. however, listContainer.innerHTML looks like the inner content of a node. We should not be saving that. It should either an array of tasks or object of tasks, converted to a string and saved in the localstorage